src/ResourceManagement/Resource/Resource.Tests/InMemoryTests/DeploymentTests.InMemory.cs

```diff
@@ -735,23 +735,6 @@ public void DeploymentTestsCancelThrowsExceptions()
         }
 
         [Fact]
-        public void DeploymentTestsWithoutModeThrowsExceptions()
```

Comment:
There were some changes in AutoRest and now the Enum types are no longer generated as nullable types. The validation on null values was also removed. As a result, this test doesn't apply now.